backend/component/masker/masker.go

```diff
@@ -776,3 +778,193 @@ func maskProtoValue(m Masker, value *structpb.Value) *structpb.Value {
 	}
 	return nil
 }
+
+func NewInnerOuterMasker(storeMaskerType storepb.MaskingAlgorithmSetting_Algorithm_InnerOuterMask_MaskType, prefixLen, suffixLen int32, substitution string) *InnerOuterMasker {
+	var maskerType int32
+	if storeMaskerType == storepb.MaskingAlgorithmSetting_Algorithm_InnerOuterMask_INNER {
+		maskerType = InnerOuterMaskerTypeInner
+	} else if storeMaskerType == storepb.MaskingAlgorithmSetting_Algorithm_InnerOuterMask_OUTER {
+		maskerType = InnerOuterMaskerTypeOuter
+	} else {
+		return nil
+	}
+
+	return &InnerOuterMasker{
+		maskerType:   int32(maskerType),
+		prefixLen:    prefixLen,
+		suffixLen:    suffixLen,
+		substitution: substitution,
+	}
+}
+
+const (
+	InnerOuterMaskerTypeInnerUndefined = 0
+	InnerOuterMaskerTypeInner          = 1
+	InnerOuterMaskerTypeOuter          = 2
+)
+
+type InnerOuterMasker struct {
+	prefixLen    int32
+	suffixLen    int32
+	maskerType   int32
```

Comment:
For Golang, we are used to defining the `maskerType` as `MaskerType`. The naming will help us to understand code better.